src/coreclr/jit/forwardsub.cpp

```diff
@@ -305,6 +311,11 @@ class ForwardSubVisitor final : public GenTreeVisitor<ForwardSubVisitor>
         return m_useFlags;
     }
 
+    ExceptionSetFlags GetExceptions() const
```

Comment:
It's probably worth additional comments here and/or on m_useExceptions/m_accumulatedExceptions that a value with >1 exceptions isn't trustworthy to avoid some accidental misuse.

Reply:
Will add that.